Topcoder/TCPhoneHomeEasy.cc

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <algorithm>
+#include <string>
+#include <vector>
+#include <set>
+using namespace std;
+class TCPhoneHome {
```

Comment:
Use +2 indentation instead of +4.
https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs